sources/platform/actors/publishing/monetize/pay_per_event.mdx

```diff
@@ -294,35 +314,158 @@ You can display a status message or push a record to the dataset to inform users
 
 If you're not using the Apify SDKs (JS/Python), you need to handle idempotency (ensuring the same operation produces the same result when called multiple times) manually to prevent charging the same event multiple times.
 
-## Example of a PPE pricing model
+## Examples of a PPE pricing model
+
+### Example: simple PPE pricing
+
+- `scraped-product`: $0.01 per product - count every product record you return.
+- `scraped-product-detail`: $0.05 per detail - count every enriched product detail you provide.
```

Comment:
Let's provide realistic examples in the Docs. The original pricing was too expensive, and "product" and "product-detail" means the same thing. Yes, it's true that you can scrape product information from a listing page, and then from a detail page, and the amount of data you retrieve from each will be different, but that's such a technical detail that I don't think it's a good choice for an example. Repeating `scraped` is also redundant. Everything is scraped.

```suggestion
- `product`: $0.002 per product detail - count every product record you return.
- `product-review`: $0.001 per review - count every product review you provide.
```